<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockMatcher</class>
 <widget class="QMainWindow" name="BlockMatcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatchBlocks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importData">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Imports textures</string>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="R">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Sets red value</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="G">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Sets green value</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="B">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Sets blue value</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="rLabel">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="gLabel">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="bLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="blockAmountShown">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>250</y>
      <width>47</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Sets the amount of blocks that are displayed in the window</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="blockAmountShownLabel">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Blocks Displayed:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayBlocks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>displays blocks whose color is similar to the RGB value entered</string>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="alphaFilterLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Filter:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="alphaFilter">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>280</y>
      <width>47</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filters blocks depending on how many of their pixels are transparent. &lt;/p&gt;&lt;p&gt;For example an alpha filter of 0.5 will only show blocks where 50% of its pixels are non transparent. &lt;/p&gt;&lt;p&gt;This is to filter out &amp;quot;blocks&amp;quot; that have transparent elements such as torches, plants, and candles.  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QListWidget" name="blockDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>552</width>
      <height>221</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="note">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note: clicking a block copies a minecraft command that gives you 10 of that block to your clipboard</string>
    </property>
   </widget>
   <widget class="QLabel" name="credit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Created by George S.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Picker</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorPicked">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>47</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <zorder>blockDisplay</zorder>
   <zorder>importData</zorder>
   <zorder>R</zorder>
   <zorder>G</zorder>
   <zorder>B</zorder>
   <zorder>rLabel</zorder>
   <zorder>gLabel</zorder>
   <zorder>bLabel</zorder>
   <zorder>blockAmountShown</zorder>
   <zorder>blockAmountShownLabel</zorder>
   <zorder>displayBlocks</zorder>
   <zorder>alphaFilterLabel</zorder>
   <zorder>alphaFilter</zorder>
   <zorder>note</zorder>
   <zorder>credit</zorder>
   <zorder>getColor</zorder>
   <zorder>colorPicked</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
